readme.md

```diff
@@ -81,6 +81,12 @@ Spring Cloud has a second application context named bootstrap that gets initiali
 
 If you still want the post processor to run twice or if you are using [spring-boot-devtools](https://docs.spring.io/spring-boot/docs/current/reference/html/using-spring-boot.html#using-boot-devtools-restart), you can set the optional property `awsParameterStorePropertySource.supportMultipleApplicationContexts` to `true`. The default property value is `false`to prevent multiple initializations. If you are also using Spring Cloud, this property will only work if set in the bootstrap properties.
 
+## Multi-region
+- Set `awsParameterStoreSource.ssmClient.signingRegions` with the regions from which you need to retrieve parameters using a **comma-separated** list such as `us-east-1,us-east-2`. It adds a PropertySource for each region specified. It will start looking from the first region and so on until it finds the property so put the regions in order of precedence.  

```

Comment:
I think we should divert from the `signingRegions` key, it's misleading IMO. This is refering to the other feature of specifying the endpoint + signingRegion and this is not the case here.

I suggest `awsParameterStoreSource.multiRegion.ssmClient.regions`. With this we'll have a new subnamespace `multiRegion` for future improvements. Thoughts?

Reply:
Yes sounds better! Thanks

Reply:
done